utils/CI/schema_validation.sh

```diff
@@ -8,7 +8,7 @@ function validate_core()
     echo "------"
     echo "Validating $NAME..."
     
-    ./wesnoth --validate data/_main.cfg &> temp.log || SUCCESS="No"
+    ./wesnoth --validate data/_main.cfg --preprocess-defines=SCHEMA_SHOULD_SKIP_THIS &> temp.log || SUCCESS="No"
```

Comment:
I'd rather call the define something like `SCHEMA_VALIDATION`.

Reply:
Done, and I also defined it in the C++ instead of in the CI script.